test/controllers/pensions/paymentsIntoPension/ReliefAtSourcePensionsControllerSpec.scala

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.pensions.paymentsIntoPension
+
+import builders.PaymentsIntoPensionVewModelBuilder.aPaymentsIntoPensionsEmptyViewModel
+import builders.PensionsUserDataBuilder.pensionsUserDataWithPaymentsIntoPensions
+import builders.UserBuilder.aUser
+import common.SessionValues
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.http.Status.{INTERNAL_SERVER_ERROR, OK}
+import play.api.mvc.AnyContentAsEmpty
+import play.api.mvc.Results.{BadRequest, InternalServerError, Redirect}
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{contentAsString, contentType, status}
+import support.mocks.{MockAuthorisedAction, MockErrorHandler, MockPensionSessionService}
+import support.{ControllerUnitTest, ViewHelper}
+import views.html.pensions.paymentsIntoPensions.ReliefAtSourcePensionsView
+import controllers.errors.routes.UnauthorisedUserErrorController
+
+
+class ReliefAtSourcePensionsControllerSpec extends ControllerUnitTest with ViewHelper
```

Comment:
I'm curious as to why we have a number of controllers in this package, but only one controller test.

Reply:
If it wasn't necessary, could we now delete it?

Or does it have some value.

To leave it might be confusing to others.

Reply:
Yeah we can delete it :)